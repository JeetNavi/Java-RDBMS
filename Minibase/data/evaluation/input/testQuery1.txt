Q(x,y,z) :- U(x, y, z), y = 9, z != 'adbs'
