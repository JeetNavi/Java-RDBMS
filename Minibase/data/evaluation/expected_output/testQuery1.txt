Tuple{values=[7, 9, 'cl']}
Tuple{values=[8, 9, 'rl']}
Tuple{values=[8, 9, 'ppls']}